spring-petclinic-microservices/pom.xml

```diff
@@ -111,6 +113,7 @@
           <plugin>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-maven-plugin</artifactId>
+            <version>${version.spring.boot}</version>
```

Comment:
Same here. Why didn't it cause failure before this PR?

Reply:
delete it.

Just make sure the version defined in root `pom.xml`:

```xml
<build>
    <plugins>
      <plugin>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-maven-plugin</artifactId>
        <version>${version.spring.boot}</version>
      </plugin>
      ...
    </plugins>
</build>
```

======

Why didn't it cause failure before this PR?

Because our project inherit from `spring-boot-starter-parent` at first, but now our project use scope import.

With maven you can only inherit pluginManagement when using that POM as a parent.
Scope import only brings you dependencyManagement. Check it [here](https://stackoverflow.com/a/40607508/4239209).

So we should use version to define `spring-boot-maven-plugin` version.